vue3-apollo-quasar/src/composables/github/useRepoBranches.ts

```diff
@@ -0,0 +1,34 @@
+import { useQuery, useResult } from '@vue/apollo-composable';
+import { REPO_BRANCHES_QUERY } from './queries';
+
+export const useRepoBranches = () => {
```

Comment:
Please add an interface/type with the return type for this function. Reference `useFormatter` for how this would be implemented